bulk.go

```diff
@@ -6,10 +6,14 @@ type bulkID struct {
 	ID string `json:"id"`
 }
 
-type BulkGet struct {
+type BulkGetReq struct {
```

Comment:
I'd try to avoid making this breaking change. It doesn't seem important enough to justify breaking usages.

Reply:
Right, thanks. eb002c8b0e86d58682421acd4f89df430c6f76b7.